custom/icds_reports/utils/aggregation_helpers/distributed/child_health_monthly.py

```diff
@@ -355,7 +373,13 @@ def _state_aggregation_query(self, state_id):
             ("birth_weight", "child_health.birth_weight"),
             ("child_person_case_id", "child_health.mother_id"),
             ("delivery_nature", "del_form.delivery_nature"),
-            ("term_days", "(del_form.add::DATE - del_form.edd::DATE) + 280")
+            ("term_days", "(del_form.add::DATE - del_form.edd::DATE) + 280"),
+            ("valid_status_daily", "CASE WHEN {} THEN 1 ELSE 0 END".format(valid_status_daily)),
+            ("migration_status_daily", "CASE WHEN {} THEN 0 ELSE 1 END".format(not_migration_status_daily)),
+            ("alive_status_daily", "CASE WHEN {} THEN 1 ELSE 0 END".format(alive_status_daily)),
+            ("duplicate_status_daily", "CASE WHEN {} AND person_cases.reason_closure in ('dupe_reg','incorrect_reg')"
+                                       " THEN 1 ELSE 0 END".format(not open_status_daily)),
```

Comment:
will this work like this? or the not should be part of string.

Reply:
changed